### ISSUE TITLE
[Bug]: Newly created folders dissapear/merge on creation

### ISSUE THREAD

**Issue by anonbergine** [nextcloud/desktop#5420]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

On Windows, with very latest NC stable client (but also the past few versions at least), when I create a folder, and then create anotehr folder alognside it soon after, the first folder is sometimes completely deleted.

I strongly suspect this is due to Windows calling new folders "New folder" so maybe it's seeing me rename the first one (upon creation) and then the second one and just thinking "hey, that single 'New folder' was renamed twice" rather than realising "A new folder was created, then renamed, then another new folder was created, then renamed".

### Steps to reproduce

In a synced folder:

1. Create a new folder - it's name `New folder` is auto-selected ready for editing (as is normal on Windows): Name it `Foo`.
2. Alongside that newly created `Foo` folder, create another new folder and name it `Bar`
3. On occasion, `Foo` folder is eradicated, leaving only `Bar`.

In some cases, subsequent renames can even end up getting merged. For example, if, after creating `Bar`, I then rename `Foo` to `Foo Snafu`, I might end up with a folder called `Bar Snafu`.

This bug has existed for at least the past 2 updates to NC desktop client.

### Expected behavior

If I creaate and name two new folders I should have two new folders with the names I gave them.

### Which files are affected by this bug

Any new folders

### Operating system

Windows

### Which version of the operating system you are running.

windows-10.0.19045

### Package

Appimage

### Nextcloud Server version

Updated to latest stable ~2 days ago

### Nextcloud Desktop Client version

3.7.1stable-Win64 (build 20230206)

### Is this bug present after an update or on a fresh install?

Updated from a minor version (ex. 3.4.2 to 3.4.4)

### Are you using the Nextcloud Server Encryption module?

Encryption is Enabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

```shell
I have no access to server logs.

As for client logs, they were filled with PII lol.
```

**Comment by anonbergine** [comment]
The only workaround I've found so far is to wait for newly created folder to fully sync before creating anotehr folder, which is tedious af.

**Comment by SamSirry** [comment]
In my case, if I create `New Folder` then rename it to `foo`, I get an error notification toast, then `New Folder` reappears. And now I have both folders.

IMO the client should be patient when it comes to sync'ing newly detected folders if their name is `New Folder`.. But then this name can appear in different languages...

Then perhaps let the NC client wait some seconds before sync'ing any newly detected folder that is *empty*.

**Comment by Sieboldianus** [comment]
I agree, the client should wait at least 5 seconds for newly created files and folders before syncing. I have the same problem for both files and folders - immediately get warnings that file/folders could not be synced, when in reality I simply created a new file/folder and renamed it. The sync client then tries again and it always works, but it is annoying to see all these sync issue popups when working in sync folders.

**Comment by anonbergine** [comment]
I also have same issue as mentioned by [SamSirry](https://github.com/SamSirry) above in that even just renaming a single new folder will often result in two folders (one called `New folder` and one under the new name). Server admin even tried adding `New folder` to exclude list to prevent uploads to server but it didn't seem to work?

**Comment by Rello** [comment]
Hello,

the client was enhanced significantly since the opening of this ticket.
You can help us with a retest.
If you are still encountering issues, please get back to us
